package.json

```diff
@@ -212,7 +212,7 @@
 		"compile": "npm run check-types && npm run lint && node esbuild.js",
 		"compile-tests": "tsc -p . --outDir out",
 		"install:all": "npm install && cd webview-ui && npm install",
-		"lint": "eslint src --ext ts && npm run lint --prefix webview-ui",
+		"lint": "eslint src --ext ts --quiet && npm run lint --prefix webview-ui",
```

Comment:
There seem to be a ton of warnings, so I can see why we'd add `--quiet`, but ideally we should treat everything as an error and fix or disable the current warnings.

Reply:
> There seem to be a ton of warnings, so I can see why we'd add `--quiet`, but ideally we should treat everything as an error and fix or disable the current warnings.

@cte  if i do that it will has a ton of change in this PR, this pr i want to just setting it up, and another PR to fix the eslint warning & error